go/utils/typeconversion.go

```diff
@@ -0,0 +1,99 @@
+package utils
+
+import (
+	"fmt"
+	"github.com/apache/arrow/go/arrow"
+	"github.com/apache/arrow/go/arrow/array"
+	"github.com/feast-dev/feast/go/protos/feast/types"
+)
+
+func ProtoTypeToArrowType(sample *types.Value) (arrow.DataType, error) {
+	switch sample.Val.(type) {
+	case *types.Value_BytesVal:
+		return arrow.FixedWidthTypes.Boolean, nil
```

Comment:
So arrow boolean maps to Feast bytes (as opposed to https://github.com/feast-dev/feast/blob/bb8274147888ba70ee50282653f2aec566672911/protos/feast/types/Value.proto#L34)

Reply:
Why not use binary or fixed sized binary for bytes?

Reply:
Fixed